numpy/_core/src/umath/ufunc_object.c

```diff
@@ -984,6 +984,10 @@ try_trivial_single_output_loop(PyArrayMethod_Context *context,
         res = -1;
     }
 
+    /*
+     * Do not use PyArray_CheckRetAndFPE since we don't want to look up the
+     * name if no error
```

Comment:
I suppose that may have been a thought here, but I wonder if that is worth it, the function is just `return ufunc->name ? ufunc->name : "<unnamed ufunc>";`?